Build/Build.cs

```diff
@@ -354,7 +354,11 @@ void ReportTestOutcome(params string[] globFilters)
         .ProceedAfterFailure()
         .Executes(() =>
         {
-            Node($"{YarnCli} --silent install", workingDirectory: RootDirectory);
+            Node($"{YarnCli} --silent install", workingDirectory: RootDirectory,
+                logger: (_, msg) =>
+                {
+                    if (!msg.Contains("warning Pattern")) Error(msg);
```

Comment:
🔧 We need some comments to explain what we're doing here. 
🙃 (nitpick) Please use curly braces here